code/modules/mob/living/simple_animal/friendly/drone/_drone.dm

```diff
@@ -27,7 +27,7 @@
 	maxHealth = 30
 	unsuitable_atmos_damage = 0
 	wander = 0
-	speed = 0
+	speed = -0.75 // Buff to make drone more bearable during time dilation etc
```

Comment:
I dont think this is nessecary. Drones can vent crawl and already seem fast, this will make them very hard to click